src/vmm_mad/remotes/lib/vcenter_driver/virtual_machine.rb

```diff
@@ -1092,6 +1200,32 @@ def set_boot_order(boot_info)
         RbVmomi::VIM.VirtualMachineBootOptions({bootOrder: boot_order})
     end
 
+    def sync_nics(option = :none, execute = true)
```

Comment:
Add a description of what this does. Are we removing all NICs? Only removing the ones missing from OpenNebula? A short, concise paragraph suffices.